### PR TITLE
Checkbox with terms accept to client apply form

### PR DESCRIPTION
**Pull request by SebaKas21**

### CODE REVIEW

File: be/src/Infrastructure/Localization/Foundation.Localization/Resources/GlobalResources.pl.resx [hunk 1]
```diff
@@ -570,4 +570,7 @@
   <data name="VicePresident" xml:space="preserve">
     <value>Wiceprezes</value>
   </data>
+  <data name="AcceptTermsText" xml:space="preserve">
+    <value>Akceptuję Regulamin i Politykę Prywatności</value>
```
Comment (dawiddworak88) [inline be/src/Infrastructure/Localization/Foundation.Localization/Resources/GlobalResources.pl.resx]: Hi @SebaKas21 ,

let's make Terms and Conditions & Privacy Policy links that lead to the terms and conditions and privacy policy pages.

Cheers,

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]

File: fe/src/project/Account/areas/Accounts/components/RegisterForm/RegisterForm.js [hunk 2]
```diff
@@ -311,13 +312,26 @@ const RegisterForm = (props) => {
                                     error={(errors.companyPostalCode.length > 0) && dirty.companyPostalCode} />
                             </div>
                         </div>
+                        <div className="field">
+                            <NoSsr>
+                                <FormControlLabel 
+                                    control={
+                                        <Checkbox 
+                                            checked={acceptedTerms}
```
Comment (dawiddworak88) [inline fe/src/project/Account/areas/Accounts/components/RegisterForm/RegisterForm.js:322]: **suggestion:** we should move it to the stateSchema

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]
We're almost there :)

File: fe/src/project/Account/areas/Accounts/components/RegisterForm/RegisterForm.js (hunk 2 above)
Comment (SebaKas21) [reply fe/src/project/Account/areas/Accounts/components/RegisterForm/RegisterForm.js:322]: okey

**Review by dawiddworak88 — APPROVED** [review]
Merci 🥇

**Pull request merged**